ui/src/components/MessageList.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { List, ListItem } from 'semantic-ui-react';
+import { Message } from '@daml2ts/create-daml-app/lib/create-daml-app-0.1.0/User';
+import { useParty, useQuery } from '@daml/react';
+
+/**
+ * React component displaying the list of messages for the current user.
+ */
+const MessageList: React.FC = () => {
+  const username = useParty();
+  const messagesResult = useQuery(Message, () => ({receiver: username}), []);
+  const messages = messagesResult.contracts.map(message => message.payload);
+
+  const showMessage = (message: Message): string => {
+    return (message.sender + ": " + message.content);
+  }
+
+  return (
+    <List relaxed>
+      {messages.map(message => <ListItem>{showMessage(message)}</ListItem>)}
```

Comment:
```suggestion
      {messages.map(({sender, content}) => <ListItem>{sender}: {content}</ListItem>)}
```
and remove the `showMesage` function above. There's no need to make the code longer than need be. We probably also need to set the `key` property of the `ListItem`s or we'll get warnings on the console.